src/Functional.Primitives.FluentAssertions/OptionTypeAssertions.cs

```diff
@@ -33,19 +35,21 @@ public OptionTypeAssertions(Option<T> subject)
 		/// <param name="because">Additional information for if the assertion fails.</param>
 		/// <param name="becauseArgs">Zero or more objects to format using the placeholders in <paramref name="because"/>.</param>
 		/// <returns></returns>
+		[CustomAssertion]
 		public AndConstraint<ObjectAssertions> Be(Option<T> expected, string because = "", params object[] becauseArgs)
 		{
 			Execute.Assertion
 				.ForCondition(_subject.Equals(expected))
 				.BecauseOf(because, becauseArgs)
+				.WithDefaultIdentifier(IDENTIFIER)
 				.FailWith(MakeFailReason);
 
 			return new AndConstraint<ObjectAssertions>(new ObjectAssertions(_subject));
 
 			FailReason MakeFailReason()
 			{
 				var builder = new StringBuilder();
-				builder.AppendLine($"Expected to be equal{{reason}}, but the two Option<{typeof(T)}> are not equal.");
+				builder.AppendLine($"Expected {{context:{IDENTIFIER}}} to be equal to the expected option{{reason}}, but the two Option<{typeof(T)}> are not equal.");
```

Comment:
I think you're missing a space in this error message, between `option` and `{{reason}}`.

Reply:
[FluentAssertions is weird like that](https://fluentassertions.com/extensibility/#building-your-own-extensions)